gtsfm/data_association/dummy_da.py

```diff
@@ -28,19 +29,21 @@ def __init__(self, reproj_error_thresh: float, min_track_len: int):
 
     def run(
         self,
+        num_images: int,
```

Comment:
i prefer we delete the `dummy_da` class at this time, as I don't believe it serves any purpose